### PR TITLE
Post Submission Actions

### PR DESCRIPTION
**Pull request by solemabrothers**
On doing the migration, some functionality was left out. This PR creates a fix for that.

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java [hunk 1]
```diff
@@ -29,7 +30,7 @@ public void applyAction(FormEntrySession session) {
             return;
         }
         ProgramWorkflowService programWorkflowService = Context.getProgramWorkflowService();
-        UgandaEMRService ugandaEMRService= Context.getService(UgandaEMRService.class);
+        UgandaEMRService ugandaEMRPOCService= Context.getService(UgandaEMRService.class);
```
Comment (slubwama) [inline omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java]: Change this to ugandaEMRService

File: omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java [hunk 2]
```diff
@@ -41,10 +42,10 @@ public void applyAction(FormEntrySession session) {
...
-            PatientProgramAttribute heathUnitTBNo=ugandaEMRService.generatePatientProgramAttributeFromObservation(currentPatientProgram, session.getEncounter().getAllObs(), TB_HEALTH_UNIT_NUMBER_CONCEPT_ID, TB_HEALTH_UNIT_NUMBER_PROGRAM_ATTRIBUTE_TYPE_UUID);
-            PatientProgramAttribute districtTBNo=ugandaEMRService.generatePatientProgramAttributeFromObservation(currentPatientProgram, session.getEncounter().getAllObs(), TB_DISTRICT_NUMBER_CONCEPT_ID, TB_DISTRICT_NUMBER_PROGRAM_ATTRIBUTE_TYPE_UUID);
-            PatientProgramAttribute drTBNo=ugandaEMRService.generatePatientProgramAttributeFromObservation(currentPatientProgram, session.getEncounter().getAllObs(), DR_TB_NUMBER_PROGRAM_CONCEPT_ID, DR_TB_NUMBER_PROGRAM_ATTRIBUTE_TYPE_UUID);
+            PatientProgramAttribute currentRegimen=ugandaEMRPOCService.generatePatientProgramAttributeFromObservation(currentPatientProgram, session.getEncounter().getAllObs(), TB_REGIMEN_CONCEPT_ID, TB_REGIMEN_PROGRAM_ATTRIBUTE_TYPE_UUID);
```
Comment (slubwama) [inline omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java]: Change this to ugandaEMRService

File: omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java [hunk 3]
```diff
@@ -41,10 +42,10 @@ public void applyAction(FormEntrySession session) {
...
+            PatientProgramAttribute heathUnitTBNo=ugandaEMRPOCService.generatePatientProgramAttributeFromObservation(currentPatientProgram, session.getEncounter().getAllObs(), TB_HEALTH_UNIT_NUMBER_CONCEPT_ID, TB_HEALTH_UNIT_NUMBER_PROGRAM_ATTRIBUTE_TYPE_UUID);
```
Comment (slubwama) [inline omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java]: Change this to ugandaEMRService

File: omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java [hunk 4]
```diff
@@ -41,10 +42,10 @@ public void applyAction(FormEntrySession session) {
...
+            PatientProgramAttribute districtTBNo=ugandaEMRPOCService.generatePatientProgramAttributeFromObservation(currentPatientProgram, session.getEncounter().getAllObs(), TB_DISTRICT_NUMBER_CONCEPT_ID, TB_DISTRICT_NUMBER_PROGRAM_ATTRIBUTE_TYPE_UUID);
```
Comment (slubwama) [inline omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java]: Change this to ugandaEMRService

File: omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java [hunk 5]
```diff
@@ -41,10 +42,10 @@ public void applyAction(FormEntrySession session) {
...
+            PatientProgramAttribute drTBNo=ugandaEMRPOCService.generatePatientProgramAttributeFromObservation(currentPatientProgram, session.getEncounter().getAllObs(), DR_TB_NUMBER_PROGRAM_CONCEPT_ID, DR_TB_NUMBER_PROGRAM_ATTRIBUTE_TYPE_UUID);
```
Comment (slubwama) [inline omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java]: Change this to ugandaEMRService

**Review by slubwama — CHANGES_REQUESTED** [review]

File: omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java (hunk 1 above)
Comment (solemabrothers) [reply omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java]: done

File: omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java (hunk 3 above)
Comment (solemabrothers) [reply omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java]: Done

File: omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java (hunk 4 above)
Comment (solemabrothers) [reply omod/src/main/java/org/openmrs/module/ugandaemr/htmlformentry/TBSummaryPageSubmissionAction.java]: Done

**Review by slubwama — APPROVED** [review]

**Pull request merged**